TFEngine.py

```diff
@@ -795,6 +798,10 @@ def init_network_from_config(self, config=None):
       except tf.errors.NotFoundError:
         print("Exiting now because model cannot be loaded.", file=log.v1)
         sys.exit(1)
+    if config.has("source_voc"):
+      self.source_voc = Vocabulary(config.value("source_voc", ""), "<UNK>")
+    if config.has("target_voc"):
+      self.target_voc = Vocabulary(config.value("target_voc", ""), "<UNK>")
```

Comment:
This must be removed here. See my other comments:
https://github.com/rwth-i6/returnn/pull/48/files/c0aa865b600430eae3d272d022def324f1964107..d5cc0cafdc3de214e0c7ef71834bf7f11d022097

Also `"<UNK>"` is a bad default. In most other cases, it was `"UNK"`. And this is already the default. This should be consistent. Also, this default must not be changed anymore, because it would break lots of setups.